src/mesh/meshblock.cpp

```diff
@@ -307,13 +307,13 @@ void MeshBlock::AllocateRealUserMeshBlockDataField(int n) {
     ATHENA_ERROR(msg);
   }
   nreal_user_meshblock_data_ = n;
-  ruser_meshblock_data = new AthenaArray<Real>[n];
+  ruser_meshblock_data = nullptr;//new ParArrayND<Real>[n];
```

Comment:
Does the `user_meshblock_data` mechanism still work? And do we want it to? At some level, containers and variables do this job better.

Reply:
I also think that the `Variables` and `Containers` provide the abstraction that was originally meant to be satisfied by the user meshblock data.
We could think about adding a "user" variable within the problem generator in one of the examples.

Reply:
removed.